updated sending-deploys (#122)
casper-network/docs-new: source/docs/casper/dapp-dev-guide/building-dapps/sending-deploys.md

# Sending Deploys to the Network

> In order to install smart contracts on the blockchain, you can send your .wasm code to the network via a [deploy](/design/casper-design.md/#execution-semantics-deploys). To do this, you will need to meet a few prerequisites:

Reviewer: ```suggestion
In order to install smart contracts on the blockchain, you can send your Wasm to the network via a [deploy](/design/casper-design.md/#execution-semantics-deploys). To do this, you will need to meet a few prerequisites:
```

- You will need a client to interact with the network, such as the [default Casper client](/dapp-dev-guide/setup#the-casper-command-line-client)
- Ensure you have an [Account](/dapp-dev-guide/setup#setting-up-an-account) and its associated [keys](/dapp-dev-guide/keys.md) This account will pay for the deploy, and its secret key will sign the deploy
- Ensure this account has enough CSPR tokens to pay for the deploy

## Paying for Deploys {#paying-for-deploys}

> CSPR tokens are used to pay for deploys on the Casper Network. There are several ways to fund your account:

Reviewer: ```suggestion
CSPR tokens are used to pay for Deploys on the Casper Network. There are several ways to fund your account:
```

4. `payment-amount` - The payment for the deploy in motes. This example uses 2.5 CSPR, but you need to modify this for your contract. See the [note](#a-note-about-gas-price) below
5. `session-path` - The path to the contract Wasm, which should point to wherever you compiled the contract (.wasm file) on your computer

> Once you call this command, it will return a deploy hash, which you will need to verify that the deploy was accepted by the node and to query the results of the deploy. Sending a deploy to the network does not mean that the transaction was processed successfully. Once the network has received the deploy and done some preliminary validation of it, it will queue up in the system before being proposed in a block for execution. Therefore, you will need to check to see that the contract was executed as expected.

Reviewer: ```suggestion
Once you execute this command, it will return a Deploy hash, which you will need to verify that the Deploy was accepted by the node and to query the execution results of the Deploy. Sending a Deploy to the network does not mean that the Deploy was processed successfully. Once the network has received the Deploy and done some preliminary validation of it, it will queue up in the system before being proposed in a block for execution. Therefore, you will need to check to see that the Deploy was executed as expected.
```
